app/index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Band Name Generator</title>
+    <style type="text/css">
```

Comment:
you can put all this in a css file and bring it in. Check out how Jerod does it:
https://github.com/inquisitivequin/band-name-generator/pull/1/files